_targets.R

```diff
@@ -0,0 +1,36 @@
+library(targets)
+
+source('1_fetch/src/fetch_model_RMSEs.R')
+source('2_process/src/process_model_RMSEs.R')
+source('3_visualize/src/create_RMSE_figure1.R')
+source('3_visualize/src/create_RMSE_table1.R')
+
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    raw_data,
+    fetch_and_load_data(out_fpath = "1_fetch/out/model_RMSEs.csv",
+                        sb_id = "5d925066e4b0c4f70d0d0599",
+                        sb_files = "me_RMSE.csv")
+  ), 
+  # Prepare the data for plotting and save it
+  tar_target(
+    eval_data,
+    process_data(raw_data = raw_data,
+                 out_fpath = "2_process/out/model_summary_results.csv"),
+  ),
+  # Create a plot
+  tar_target(
+    figure_1_png,
+    plot_RMSEs(eval_data = eval_data,
+               out_fpath = "3_visualize/out/RMSE_figure.png")
+  ),
+  # Save the model diagnostics
+  tar_target(
+    model_diagnostic_text_txt,
+    tabularize_RMSEs(eval_data = eval_data,
+                     out_fpath = "3_visualize/out/model_diagnostic_text.txt")
+  )
```

Comment:
Any time you create a file target, there are two key parts: 

1. The function _**must**_ return the filepath (so in this case, you should edit `process_data()` to have `return(out_fpath)` at the end.
2. Your call to `tar_target()` should have `format='file'` specified. That detail is buried in https://github.com/jdiaz4302/ds-pipelines-targets-1/issues/6

Those changes are needed for any target that you are making that writes a file.